<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My_3d_lab[*]</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionAdd_artist"/>
    <addaction name="actionAdd_album"/>
    <addaction name="actionAdd_track"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionRu"/>
    <addaction name="actionEn"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoOpen-48.png</normaloff>:/new/prefix1/icons/icoOpen-48.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoSave-48.png</normaloff>:/new/prefix1/icons/icoSave-48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoExit-48.png</normaloff>:/new/prefix1/icons/icoExit-48.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoEdit-48.png</normaloff>:/new/prefix1/icons/icoEdit-48.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAdd_artist">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoAddArtist-50.png</normaloff>:/new/prefix1/icons/icoAddArtist-50.png</iconset>
   </property>
   <property name="text">
    <string>Add artist</string>
   </property>
  </action>
  <action name="actionAdd_album">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoAddAlbum-50.png</normaloff>:/new/prefix1/icons/icoAddAlbum-50.png</iconset>
   </property>
   <property name="text">
    <string>Add album</string>
   </property>
  </action>
  <action name="actionAdd_track">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoAddTrack-50.png</normaloff>:/new/prefix1/icons/icoAddTrack-50.png</iconset>
   </property>
   <property name="text">
    <string>Add track</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/icons/icoDelete-50.png</normaloff>:/new/prefix1/icons/icoDelete-50.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionChange_language">
   <property name="text">
    <string>Change language ru/en</string>
   </property>
  </action>
  <action name="actionRu">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEn">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
